static/data/2017/10/events.json

```diff
@@ -161,4 +161,13 @@
     "website": "http://hackdtu.tech/",
     "logo": "http://hackdtu.tech/static/images/logo.png"
   }
```

Comment:
Add a `,` here, this is a JSON we need it here.

Reply:
Fixed it, sorry I'm kinda new to JSON!

Reply:
The comma is still missing please add it.